tests/acceptance/editor-key-commands-test.js

```diff
@@ -140,6 +141,58 @@ testStatefulCommand({
   markupName: 'em'
 });
 
+test(`cmd-left goes to the beginning of a line (MacOS only)`, (assert) => {
+  let initialText = 'something';
+  const mobiledoc = Helpers.mobiledoc.build(
+    ({post, markupSection, marker}) => post([
+      markupSection('p', [marker(initialText)])
+    ]));
+
+  editor = new Editor({mobiledoc});
+  editor.render(editorElement);
+
+  let textElement = editor.post.sections.head.markers.head.renderNode.element;
+
+  Helpers.dom.moveCursorTo(editor, textElement, 4);
+  let originalCursorPosition = Helpers.dom.getCursorPosition();
+  Helpers.dom.triggerKeyCommand(editor, 'LEFT', MODIFIERS.META);
+
+  let changedCursorPosition = Helpers.dom.getCursorPosition();
+  let expectedCursorPosition = 0; // beginning of text
+
+  if (Browser.isMac()) {
+    assert.equal(changedCursorPosition.offset, expectedCursorPosition, 'cursor moved to the beginning of the line on MacOS');
+  } else {
+    assert.equal(changedCursorPosition.offset, originalCursorPosition.offset, 'cursor not moved to the end of the line (non-MacOS)');
+  }
+});
+
+test(`cmd-right goes to the end of a line (MacOS only)`, (assert) => {
+  let initialText = 'something';
+  const mobiledoc = Helpers.mobiledoc.build(
+    ({post, markupSection, marker}) => post([
+      markupSection('p', [marker(initialText)])
+    ]));
+
+  editor = new Editor({mobiledoc});
+  editor.render(editorElement);
+
+  let textElement = editor.post.sections.head.markers.head.renderNode.element;
+
+  Helpers.dom.moveCursorTo(editor, textElement, 4);
+  let originalCursorPosition = Helpers.dom.getCursorPosition();
+  Helpers.dom.triggerKeyCommand(editor, 'RIGHT', MODIFIERS.META);
+
+  let changedCursorPosition = Helpers.dom.getCursorPosition();
+  let expectedCursorPosition = initialText.length; // end of text
+
+  if (Browser.isMac()) {
+    assert.equal(changedCursorPosition.offset, expectedCursorPosition, 'cursor moved to the end of the line on MacOS');
+  } else {
+    assert.equal(changedCursorPosition.offset, originalCursorPosition.offset, 'cursor not moved to the end of the line (non-MacOS)');
```

Comment:
So I'm wondering about that because for me, Command on windows is the Windows Key, and Windows Key Left/Right actually are OS-level commands that rearrange your windows.